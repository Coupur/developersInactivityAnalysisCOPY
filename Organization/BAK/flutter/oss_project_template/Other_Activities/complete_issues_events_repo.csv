id,date,event,creator_login
442183257,2015-10-22 01:56:11,closed,sethladd
442183256,2015-10-22 01:56:11,merged,sethladd
442183255,2015-10-22 01:56:11,referenced,sethladd
440733634,2015-10-20 19:23:46,subscribed,sethladd
440733633,2015-10-20 19:23:46,mentioned,sethladd
539989821,2016-02-04 23:54:14,closed,chinmaygarde
539989820,2016-02-04 23:54:14,merged,chinmaygarde
539989819,2016-02-04 23:54:14,referenced,chinmaygarde
555499518,2016-02-18 18:35:58,closed,collinjackson
555499516,2016-02-18 18:35:58,merged,collinjackson
555499515,2016-02-18 18:35:58,referenced,collinjackson
1068922753,2017-05-04 13:30:59,closed,mit-mit
1052220402,2017-04-21 10:37:41,subscribed,sethladd
1052220401,2017-04-21 10:37:41,mentioned,sethladd
1052220399,2017-04-21 10:37:41,subscribed,Hixie
1052220398,2017-04-21 10:37:41,mentioned,Hixie
1048187528,2017-04-19 09:22:06,subscribed,Hixie
1048187526,2017-04-19 09:22:06,mentioned,Hixie
1048061151,2017-04-19 08:02:38,subscribed,sethladd
1048061149,2017-04-19 08:02:38,mentioned,sethladd
1049313245,2017-04-19 20:24:41,subscribed,Hixie
1049313244,2017-04-19 20:24:41,mentioned,Hixie
1248091445,2017-09-13 21:12:46,head_ref_deleted,sethladd
1248091402,2017-09-13 21:12:44,closed,sethladd
1248091397,2017-09-13 21:12:44,merged,sethladd
1248091395,2017-09-13 21:12:44,referenced,sethladd
1247732629,2017-09-13 17:33:55,subscribed,Hixie
1247732628,2017-09-13 17:33:55,mentioned,Hixie
