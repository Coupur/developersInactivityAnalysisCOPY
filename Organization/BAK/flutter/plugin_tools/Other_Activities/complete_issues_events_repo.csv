id,date,event,creator_login
1124132103,2017-06-14 21:56:44,referenced,goderbauer
1121901565,2017-06-13 18:09:36,referenced,mehmetf
1083180597,2017-05-15 22:53:44,closed,goderbauer
1083180596,2017-05-15 22:53:44,merged,goderbauer
1083180595,2017-05-15 22:53:44,referenced,goderbauer
1082971374,2017-05-15 20:19:15,head_ref_force_pushed,goderbauer
1082650092,2017-05-15 17:02:45,subscribed,mit-mit
1082650091,2017-05-15 17:02:45,mentioned,mit-mit
1082650089,2017-05-15 17:02:45,subscribed,tvolkert
1082650087,2017-05-15 17:02:45,mentioned,tvolkert
1082650086,2017-05-15 17:02:45,subscribed,goderbauer
1082650085,2017-05-15 17:02:45,mentioned,goderbauer
1080787849,2017-05-12 23:25:48,head_ref_force_pushed,goderbauer
1080786544,2017-05-12 23:23:35,head_ref_force_pushed,goderbauer
1080777129,2017-05-12 23:09:07,head_ref_force_pushed,goderbauer
1080766292,2017-05-12 22:54:54,head_ref_force_pushed,goderbauer
1080765123,2017-05-12 22:53:17,head_ref_force_pushed,goderbauer
1080763205,2017-05-12 22:50:24,review_requested,goderbauer
1080763151,2017-05-12 22:50:19,review_requested,goderbauer
1080760638,2017-05-12 22:47:17,head_ref_force_pushed,goderbauer
1080758492,2017-05-12 22:44:21,head_ref_force_pushed,goderbauer
1096121683,2017-05-24 18:31:40,head_ref_deleted,goderbauer
1096121568,2017-05-24 18:31:37,closed,goderbauer
1096121564,2017-05-24 18:31:37,merged,goderbauer
1096121559,2017-05-24 18:31:37,referenced,goderbauer
1096109751,2017-05-24 18:24:11,review_requested,goderbauer
1112212664,2017-06-06 17:21:38,closed,mehmetf
1112212661,2017-06-06 17:21:38,subscribed,goderbauer
1112212659,2017-06-06 17:21:38,mentioned,goderbauer
1111063212,2017-06-06 04:04:04,subscribed,goderbauer
1111063211,2017-06-06 04:04:04,mentioned,goderbauer
1116365200,2017-06-08 22:23:01,closed,goderbauer
1116365199,2017-06-08 22:23:01,merged,goderbauer
1116365198,2017-06-08 22:23:01,referenced,goderbauer
1116362754,2017-06-08 22:20:59,review_requested,goderbauer
1124132088,2017-06-14 21:56:43,closed,goderbauer
1124132086,2017-06-14 21:56:43,merged,goderbauer
1124132085,2017-06-14 21:56:43,referenced,goderbauer
1121858558,2017-06-13 17:43:58,subscribed,tvolkert
1121858557,2017-06-13 17:43:58,mentioned,tvolkert
1124140000,2017-06-14 22:03:24,head_ref_deleted,goderbauer
1124139969,2017-06-14 22:03:22,closed,goderbauer
1124139968,2017-06-14 22:03:22,merged,goderbauer
1124139967,2017-06-14 22:03:22,referenced,goderbauer
1124137930,2017-06-14 22:01:39,subscribed,mehmetf
1124137928,2017-06-14 22:01:39,mentioned,mehmetf
1151449667,2017-07-05 18:32:38,head_ref_deleted,goderbauer
1151449595,2017-07-05 18:32:35,closed,goderbauer
1151449593,2017-07-05 18:32:35,merged,goderbauer
1151449592,2017-07-05 18:32:35,referenced,goderbauer
1151358393,2017-07-05 17:35:56,review_requested,goderbauer
1154801396,2017-07-07 16:46:11,head_ref_deleted,goderbauer
1154801315,2017-07-07 16:46:08,closed,goderbauer
1154801312,2017-07-07 16:46:08,merged,goderbauer
1154801310,2017-07-07 16:46:08,referenced,goderbauer
1153305909,2017-07-06 18:54:49,review_requested,goderbauer
1177471607,2017-07-25 07:41:00,head_ref_deleted,goderbauer
1177471497,2017-07-25 07:40:56,closed,goderbauer
1177471496,2017-07-25 07:40:56,merged,goderbauer
1177471495,2017-07-25 07:40:56,referenced,goderbauer
1176494202,2017-07-24 16:50:41,review_requested,goderbauer
1176491545,2017-07-24 16:48:59,review_requested,goderbauer
1459219840,2018-02-06 02:02:58,closed,goderbauer
1459211358,2018-02-06 01:54:10,subscribed,collinjackson
1459211357,2018-02-06 01:54:10,mentioned,collinjackson
1459209104,2018-02-06 01:51:36,head_ref_force_pushed,goderbauer
1571916977,2018-04-12 20:36:40,head_ref_deleted,mravn-google
1571914263,2018-04-12 20:35:12,closed,goderbauer
1571914261,2018-04-12 20:35:11,merged,goderbauer
1571914259,2018-04-12 20:35:11,referenced,goderbauer
1604071463,2018-05-01 20:13:30,head_ref_deleted,goderbauer
1604071405,2018-05-01 20:13:27,closed,goderbauer
1604071402,2018-05-01 20:13:27,merged,goderbauer
1604071401,2018-05-01 20:13:27,referenced,goderbauer
1604039632,2018-05-01 19:54:12,subscribed,mravn-google
1604039629,2018-05-01 19:54:12,mentioned,mravn-google
1683738595,2018-06-15 15:58:50,closed,goderbauer
1683738589,2018-06-15 15:58:50,merged,goderbauer
1683738588,2018-06-15 15:58:50,referenced,goderbauer
1684331187,2018-06-15 21:52:55,head_ref_deleted,mravn-google
1684329111,2018-06-15 21:51:14,closed,mravn-google
1684329110,2018-06-15 21:51:14,merged,mravn-google
1684329108,2018-06-15 21:51:14,referenced,mravn-google
1684297894,2018-06-15 21:26:58,subscribed,goderbauer
1684297893,2018-06-15 21:26:58,mentioned,goderbauer
1689729857,2018-06-19 21:25:24,closed,gspencergoog
1689729851,2018-06-19 21:25:24,merged,gspencergoog
1689729840,2018-06-19 21:25:23,referenced,gspencergoog
1689720445,2018-06-19 21:19:59,head_ref_force_pushed,gspencergoog
1689625242,2018-06-19 20:32:32,head_ref_force_pushed,gspencergoog
1689516316,2018-06-19 19:38:50,review_requested,gspencergoog
1684327345,2018-06-15 21:49:41,head_ref_force_pushed,gspencergoog
1684306265,2018-06-15 21:33:25,head_ref_force_pushed,gspencergoog
1684306057,2018-06-15 21:33:13,head_ref_force_pushed,gspencergoog
1684739742,2018-06-16 18:38:26,head_ref_deleted,mravn-google
1684739430,2018-06-16 18:37:08,closed,mravn-google
1684739429,2018-06-16 18:37:08,merged,mravn-google
1684739428,2018-06-16 18:37:08,referenced,mravn-google
1684741529,2018-06-16 18:44:58,head_ref_deleted,mravn-google
1684741509,2018-06-16 18:44:54,closed,mravn-google
1684741506,2018-06-16 18:44:54,merged,mravn-google
1684741505,2018-06-16 18:44:54,referenced,mravn-google
1684741344,2018-06-16 18:44:25,review_requested,mravn-google
1684740948,2018-06-16 18:42:52,head_ref_force_pushed,mravn-google
1733094163,2018-07-13 17:36:15,closed,mit-mit
1733094161,2018-07-13 17:36:15,merged,mit-mit
1733094160,2018-07-13 17:36:15,referenced,mit-mit
1732387579,2018-07-13 11:34:30,review_requested,mit-mit
1733539246,2018-07-13 22:04:15,head_ref_deleted,goderbauer
1733529578,2018-07-13 21:57:30,closed,goderbauer
1733529574,2018-07-13 21:57:30,merged,goderbauer
1733529572,2018-07-13 21:57:30,referenced,goderbauer
1733518177,2018-07-13 21:48:41,review_requested,goderbauer
1736154056,2018-07-16 20:11:31,closed,gspencergoog
1736154051,2018-07-16 20:11:31,merged,gspencergoog
1736154049,2018-07-16 20:11:31,referenced,gspencergoog
1733721842,2018-07-14 03:23:06,review_requested,gspencergoog
1736470267,2018-07-16 23:27:17,closed,gspencergoog
1736470265,2018-07-16 23:27:17,merged,gspencergoog
1736470264,2018-07-16 23:27:16,referenced,gspencergoog
1736448525,2018-07-16 23:07:36,review_requested,gspencergoog
1736415352,2018-07-16 22:41:27,head_ref_force_pushed,gspencergoog
1913535315,2018-10-18 22:49:07,closed,dnfield
1913535311,2018-10-18 22:49:07,merged,dnfield
1913535309,2018-10-18 22:49:07,referenced,dnfield
1901099864,2018-10-12 15:52:47,subscribed,mravn-google
1901099862,2018-10-12 15:52:47,mentioned,mravn-google
1913563731,2018-10-18 23:11:19,closed,dnfield
1913563729,2018-10-18 23:11:19,merged,dnfield
1913563728,2018-10-18 23:11:19,referenced,dnfield
1913532044,2018-10-18 22:46:30,review_requested,dnfield
1913662698,2018-10-19 00:45:04,closed,dnfield
1913662697,2018-10-19 00:45:04,merged,dnfield
1913662696,2018-10-19 00:45:04,referenced,dnfield
1913579821,2018-10-18 23:25:04,review_requested,dnfield
1915565214,2018-10-19 19:30:09,closed,dnfield
1915565213,2018-10-19 19:30:09,merged,dnfield
1915565211,2018-10-19 19:30:09,referenced,dnfield
1915449727,2018-10-19 18:29:28,review_requested,dnfield
1915572204,2018-10-19 19:34:18,closed,gspencergoog
1915572199,2018-10-19 19:34:18,merged,gspencergoog
1915572198,2018-10-19 19:34:18,referenced,gspencergoog
1915569340,2018-10-19 19:32:42,review_requested,dnfield
2211601906,2019-03-18 20:07:06,referenced,collinjackson
2205523126,2019-03-15 05:45:36,head_ref_deleted,collinjackson
2205523053,2019-03-15 05:45:32,closed,collinjackson
2205523049,2019-03-15 05:45:32,merged,collinjackson
2205523047,2019-03-15 05:45:32,referenced,collinjackson
2205340975,2019-03-15 02:50:53,renamed,collinjackson
2205340885,2019-03-15 02:50:47,review_requested,collinjackson
2205340722,2019-03-15 02:50:36,review_requested,collinjackson
2205323805,2019-03-15 02:36:05,renamed,collinjackson
2205548112,2019-03-15 06:06:53,review_requested,collinjackson
2205530196,2019-03-15 05:51:45,review_requested,collinjackson
2264364960,2019-04-09 22:06:17,head_ref_deleted,collinjackson
2264364786,2019-04-09 22:06:12,closed,collinjackson
2264364771,2019-04-09 22:06:12,merged,collinjackson
2264364769,2019-04-09 22:06:12,referenced,collinjackson
2452234945,2019-07-01 19:35:21,subscribed,amirh
2452234944,2019-07-01 19:35:21,mentioned,amirh
2466807556,2019-07-08 16:51:48,head_ref_deleted,collinjackson
2466768191,2019-07-08 16:36:48,closed,collinjackson
2466768184,2019-07-08 16:36:48,merged,collinjackson
2466768182,2019-07-08 16:36:48,referenced,collinjackson
2466723772,2019-07-08 16:20:18,review_requested,collinjackson
2466723564,2019-07-08 16:20:13,review_requested,collinjackson
2466870450,2019-07-08 17:15:26,head_ref_deleted,collinjackson
2466870290,2019-07-08 17:15:22,closed,collinjackson
2466870276,2019-07-08 17:15:22,merged,collinjackson
2466870274,2019-07-08 17:15:22,referenced,collinjackson
2466862565,2019-07-08 17:12:14,renamed,collinjackson
2467408256,2019-07-08 20:52:24,closed,collinjackson
2467408250,2019-07-08 20:52:24,merged,collinjackson
2467408247,2019-07-08 20:52:24,referenced,collinjackson
2470459417,2019-07-09 19:38:23,head_ref_deleted,collinjackson
2470459162,2019-07-09 19:38:16,closed,collinjackson
2470459153,2019-07-09 19:38:16,merged,collinjackson
2470459151,2019-07-09 19:38:16,referenced,collinjackson
2467901060,2019-07-09 02:14:19,review_requested,collinjackson
2467865870,2019-07-09 01:44:45,head_ref_force_pushed,collinjackson
2497664895,2019-07-20 17:37:25,review_requested,collinjackson
2497664859,2019-07-20 17:37:20,closed,collinjackson
2497664858,2019-07-20 17:37:20,merged,collinjackson
2497664854,2019-07-20 17:37:20,referenced,collinjackson
