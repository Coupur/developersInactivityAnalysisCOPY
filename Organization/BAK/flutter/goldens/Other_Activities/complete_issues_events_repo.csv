id,date,event,creator_login
1662455741,2018-06-04 22:53:14,closed,HansMuller
1662455740,2018-06-04 22:53:14,merged,HansMuller
1662455737,2018-06-04 22:53:14,referenced,HansMuller
1663877927,2018-06-05 14:36:24,closed,HansMuller
1663877919,2018-06-05 14:36:24,merged,HansMuller
1663877916,2018-06-05 14:36:24,referenced,HansMuller
1726441924,2018-07-10 20:12:16,head_ref_deleted,cbracken
1726441867,2018-07-10 20:12:14,head_ref_deleted,cbracken
1714993881,2018-07-03 20:54:43,review_requested,goderbauer
1721017330,2018-07-06 20:23:15,head_ref_deleted,aam
1721017181,2018-07-06 20:23:11,closed,aam
1721017176,2018-07-06 20:23:10,merged,aam
1721017173,2018-07-06 20:23:10,referenced,aam
1721010678,2018-07-06 20:19:07,review_requested,aam
1721010541,2018-07-06 20:19:01,review_requested,aam
1726446550,2018-07-10 20:14:43,referenced,cbracken
1726250802,2018-07-10 18:31:19,referenced,cbracken
1724475895,2018-07-10 02:08:16,head_ref_deleted,aam
1724475830,2018-07-10 02:08:10,closed,aam
1724475824,2018-07-10 02:08:10,merged,aam
1724475822,2018-07-10 02:08:10,referenced,aam
1724456765,2018-07-10 01:47:46,review_requested,aam
1724447462,2018-07-10 01:37:37,review_requested,aam
1726442193,2018-07-10 20:12:23,head_ref_deleted,cbracken
1726238823,2018-07-10 18:25:33,head_ref_force_pushed,cbracken
1726238821,2018-07-10 18:25:33,closed,cbracken
1726441473,2018-07-10 20:12:02,head_ref_deleted,cbracken
1726441357,2018-07-10 20:11:57,closed,cbracken
1726441355,2018-07-10 20:11:57,merged,cbracken
1726441354,2018-07-10 20:11:57,referenced,cbracken
1726426533,2018-07-10 20:04:09,subscribed,aam
1726426532,2018-07-10 20:04:09,mentioned,aam
1728648880,2018-07-11 18:36:48,head_ref_deleted,cbracken
1728648720,2018-07-11 18:36:44,closed,cbracken
1728648719,2018-07-11 18:36:44,merged,cbracken
1728648718,2018-07-11 18:36:44,referenced,cbracken
1768906640,2018-08-03 00:05:20,head_ref_deleted,amirh
1768906601,2018-08-03 00:05:17,closed,amirh
1768906595,2018-08-03 00:05:17,merged,amirh
1768906594,2018-08-03 00:05:17,referenced,amirh
1831682936,2018-09-06 23:29:49,head_ref_deleted,GaryQian
1813655872,2018-08-28 17:11:54,closed,jacob314
1816701342,2018-08-29 20:59:05,closed,jacob314
1816701339,2018-08-29 20:59:05,merged,jacob314
1816701337,2018-08-29 20:59:05,referenced,jacob314
1814748094,2018-08-29 05:50:55,head_ref_force_pushed,jacob314
1828584899,2018-09-05 18:26:51,head_ref_deleted,jacob314
1828584758,2018-09-05 18:26:47,closed,jacob314
1828584749,2018-09-05 18:26:47,merged,jacob314
1828584746,2018-09-05 18:26:47,referenced,jacob314
1828528578,2018-09-05 18:00:37,review_requested,jacob314
1838063522,2018-09-11 09:39:04,head_ref_deleted,goderbauer
1838063419,2018-09-11 09:39:01,closed,goderbauer
1838063413,2018-09-11 09:39:01,merged,goderbauer
1838063412,2018-09-11 09:39:01,referenced,goderbauer
1838062886,2018-09-11 09:38:47,head_ref_force_pushed,goderbauer
1840855816,2018-09-12 11:18:00,head_ref_deleted,goderbauer
1840855732,2018-09-12 11:17:57,closed,goderbauer
1840855719,2018-09-12 11:17:57,merged,goderbauer
1840855717,2018-09-12 11:17:57,referenced,goderbauer
1840902474,2018-09-12 11:41:51,head_ref_deleted,goderbauer
1840902372,2018-09-12 11:41:48,closed,goderbauer
1840902371,2018-09-12 11:41:48,merged,goderbauer
1840902369,2018-09-12 11:41:48,referenced,goderbauer
1946802947,2018-11-05 17:04:24,closed,tvolkert
1946802941,2018-11-05 17:04:24,merged,tvolkert
1946802938,2018-11-05 17:04:24,referenced,tvolkert
1947537266,2018-11-05 22:24:16,head_ref_deleted,goderbauer
1947537160,2018-11-05 22:24:13,closed,goderbauer
1947537158,2018-11-05 22:24:13,merged,goderbauer
1947537156,2018-11-05 22:24:13,referenced,goderbauer
1950231727,2018-11-06 22:09:12,closed,tvolkert
1950231724,2018-11-06 22:09:12,merged,tvolkert
1950231722,2018-11-06 22:09:12,referenced,tvolkert
2076320675,2019-01-16 02:57:36,head_ref_deleted,goderbauer
2076320626,2019-01-16 02:57:34,closed,goderbauer
2076320624,2019-01-16 02:57:33,merged,goderbauer
2076320623,2019-01-16 02:57:33,referenced,goderbauer
2064314180,2019-01-09 21:56:00,closed,dnfield
2064314172,2019-01-09 21:56:00,merged,dnfield
2064314169,2019-01-09 21:56:00,referenced,dnfield
2090275680,2019-01-22 23:43:04,closed,jslavitz
2090275675,2019-01-22 23:43:04,merged,jslavitz
2090275673,2019-01-22 23:43:04,referenced,jslavitz
2084243579,2019-01-19 00:44:10,head_ref_force_pushed,jslavitz
2084241522,2019-01-19 00:42:04,head_ref_force_pushed,jslavitz
2090360881,2019-01-23 00:45:46,closed,jslavitz
2090360876,2019-01-23 00:45:45,merged,jslavitz
2090360875,2019-01-23 00:45:45,referenced,jslavitz
2102436401,2019-01-29 01:51:01,closed,jslavitz
2102436399,2019-01-29 01:51:01,merged,jslavitz
2102436396,2019-01-29 01:51:01,referenced,jslavitz
2153332746,2019-02-20 22:07:31,head_ref_deleted,xster
2153332645,2019-02-20 22:07:29,closed,xster
2153332643,2019-02-20 22:07:29,merged,xster
2153332642,2019-02-20 22:07:29,referenced,xster
2182912135,2019-03-06 02:49:51,head_ref_force_pushed,jslavitz
2182908595,2019-03-06 02:47:13,head_ref_force_pushed,jslavitz
2159057146,2019-02-22 21:55:41,head_ref_force_pushed,jslavitz
2159049941,2019-02-22 21:51:53,head_ref_force_pushed,jslavitz
2298484886,2019-04-24 20:59:20,referenced,dnfield
2298483425,2019-04-24 20:58:43,referenced,dnfield
2295672596,2019-04-24 00:05:51,head_ref_deleted,xster
2295672536,2019-04-24 00:05:48,closed,xster
2295672535,2019-04-24 00:05:48,merged,xster
2295672532,2019-04-24 00:05:48,referenced,xster
2298484979,2019-04-24 20:59:22,head_ref_deleted,dnfield
2298484864,2019-04-24 20:59:19,closed,dnfield
2298484861,2019-04-24 20:59:19,merged,dnfield
2298484860,2019-04-24 20:59:19,referenced,dnfield
2298484665,2019-04-24 20:59:14,subscribed,xster
2298484664,2019-04-24 20:59:14,mentioned,xster
2298825459,2019-04-25 00:32:46,head_ref_deleted,xster
2298825425,2019-04-25 00:32:43,closed,xster
2298825422,2019-04-25 00:32:42,merged,xster
2298825420,2019-04-25 00:32:42,referenced,xster
2357708812,2019-05-21 23:23:21,head_ref_deleted,justinmc
2357708743,2019-05-21 23:23:18,closed,justinmc
2357708738,2019-05-21 23:23:18,merged,justinmc
2357708736,2019-05-21 23:23:18,referenced,justinmc
2357410940,2019-05-21 20:45:51,head_ref_force_pushed,justinmc
2357409623,2019-05-21 20:45:23,head_ref_force_pushed,justinmc
2357406651,2019-05-21 20:44:19,head_ref_force_pushed,justinmc
2357314962,2019-05-21 20:08:22,subscribed,GaryQian
2357314961,2019-05-21 20:08:22,mentioned,GaryQian
2357313086,2019-05-21 20:07:37,head_ref_force_pushed,justinmc
2357283654,2019-05-21 19:55:37,review_requested,justinmc
2357003977,2019-05-21 18:03:24,assigned,justinmc
2357003968,2019-05-21 18:03:23,review_requested,justinmc
2359865510,2019-05-22 16:07:26,head_ref_deleted,justinmc
2359865293,2019-05-22 16:07:22,closed,justinmc
2359865271,2019-05-22 16:07:22,merged,justinmc
2359865264,2019-05-22 16:07:22,referenced,justinmc
2359842350,2019-05-22 15:59:37,assigned,justinmc
2359842339,2019-05-22 15:59:37,review_requested,justinmc
2359842338,2019-05-22 15:59:37,review_requested,justinmc
2470075435,2019-07-09 17:08:35,head_ref_deleted,Piinks
2386471223,2019-06-03 22:52:29,referenced,Piinks
2386458833,2019-06-03 22:44:21,closed,Piinks
2386458824,2019-06-03 22:44:20,merged,Piinks
2386458823,2019-06-03 22:44:20,referenced,Piinks
